pkgs/by-name/fu/fuzzdb/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation  {
+  pname = "fuzzdb";
+  version = "0-unstable-2020-02-26";
+
+  src = fetchFromGitHub {
+    owner = "fuzzdb-project";
+    repo = "fuzzdb";
+    rev = "5656ab25dc6bb43bae32236fab775658a90d7380";
+    hash = "sha256-7AORrXi443+VK5lbgcjqW4QS7asbXu/dCKj8uCMC0PY=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/wordlists/fuzzdb
+    cp -a * $out/share/wordlists/fuzzdb
```

Let's maybe keep `docs` and `web-backdoors` in another directory. It's still a bit grainy with some docs laying around here and there, but it's at least an attempt at separating the wordlists from the rest.

```suggestion
    mkdir -p $out/share/{fuzzdb,wordlists/fuzzdb}
    mv docs web-backdoors $out/share/fuzzdb
    mv */ $out/share/wordlists/fuzzdb
```

---

@h7x4 not sure about moving web backdoors, as this is supposed to be more of a mix of both.
Preferred would be to either have both wordlists and payloads from fuzzdb in the same directory, or to seperate them logically via `payloads` and `wordlists`.

For the docs, makes sense.

---

could the project lie in `share/fuzzdb`, and the wordlists be symlinked to `share/wordlists/fuzzdb`?

---

Yes, I think that's a good idea.